src/elm/arithmetic.coffee

```diff
@@ -243,18 +274,38 @@ module.exports.Successor = class Successor extends Expression
 
   exec: (ctx) ->
     arg = @execArgs(ctx)
+    successor = null
     if (not arg?)
       null
     else
-      MathUtil.successor arg
+      try
+        # MathUtil.successor throws on underflow, and the exception is used in
+        # the logic for evaluating `meets`
```

Comment:
If this is the case, shouldn't you be re-throwing the exception below?

Reply:
The comment was meant to indicate why we are `try..catch`ing rather than just adjusting `MathUtil.successor` to return `null` instead of throwing.  I'll update the comment to hopefully be clearer

Reply:
Oh right.  I get it now.  The other code counts on `MathUtil.successor` throwing, _not_ *this* code throwing...  Sorry for the mixup!